### DISCUSSION TITLE
Why there is no uninstallation guide in Docs.

### DISCUSSION THREAD

**Discussion by Sajiyah-Salat** [meshery/meshery#6779] [Ideas]
I have installed meshery and i have just 8gb of ram. My computer is not working fine after installation. Can You please guide me how to uninstall Meshery. Also It would help many other users So can w add that in docs.

**Comment by akshayamadhuri** [comment]
@Sajiyah-Salat  you can check this out  https://github.com/meshery/meshery/blob/master/install/README.md
